esi/lib/networks.py

```diff
@@ -112,3 +112,27 @@ def get_networks_from_port(connection, port, networks_dict={}, floating_ips_dict
         floating_network = connection.network.get_network(floating_network_id)
 
     return parent_network, trunk_networks, trunk_ports, floating_network
+
+
+def create_port(connection, node_name, network, **kwargs):
```

Comment:
I do like the unification of the methods here! I'm a bit more skeptical of the `kwargs` right now, unless we know we need it. This code means that ports created through the UI don't have device_owner set to baremetal:none, correct? That should be at the very least the default for all ports used for Ironic.

Reply:
Thank you! I’ve ensured that device_owner='baremetal:none' is set as the default specifically for ports created for Ironic use, as you noted. 
For UI-based port creation, I’ve left it out since device_owner isn’t needed there. This way, we keep it flexible for Ironic requirements without imposing it on UI-created ports.

Reply:
But doesn't the UI call this method directly? Those ports are needed for Ironic. I think it's okay for this method to be Ironic-specific, since it's an ESI sdk and ESI is Ironic-specific.